.github/workflows/refresh-lockfiles.yml

```diff
@@ -108,7 +108,7 @@ jobs:
           rm -r artifacts
         
       - name: Create Pull Request
-        uses: peter-evans/create-pull-request@052fc72b4198ba9fbc81b818c6e1859f747d49a8
+        uses: peter-evans/create-pull-request@f22a7da129c901513876a2380e2dae9f8e145330
```

Comment:
`dependabot` is clever enough to use a commit sha for third-party GHAs.... very neat :open_mouth: